_docs/tasks/istio-auth.md

```diff
@@ -9,89 +9,100 @@ type: markdown
+Only a single Istio CA is present in a Kubernetes cluster,
+which is always deployed in a dedicated namespace. The Istio CA issues certificates and keys to
+all pods in the Kubernetes cluster. It offers the following benefits:
 
-In real world systems, only a single Istio CA should be present in a Kubernetes cluster,
-which is always deployed in a dedicated namespace. The Istio CA issues certificates/keys to
-all pods in the Kubernetes cluster. This offers strong security and automatic trust between namespaces in the same cluster.
-However, this task also instructs how to deploy a namespace-scoped Istio CA,
-for easy setup and clean up during the experiments.
+* In the near future, Istio will enforce the dedicated namespace to use
+[Kubernetes RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) (beta in Kubernetes V1.6) to provide security
+boundary. This will offer strong security for Istio CA.
 
-## Enabling Istio Auth
+* Services in the same Kubernetes cluster but different namespaces are able to talk to each other through Istio Auth
+without extra trust setup.
 
-### Option 1: using per-namespace CA
+**Per-namespace CA**:
 
-Per namespace CA is convenient for doing experiments.
+[The Istio installation guide]({{home}}/docs/tasks/installing-istio.html#installing-on-an-existing-cluster)
+with "Install Istio and enable Istio Auth feature" in step 4 instructs enabling Istio Auth with per-namespace CA.
+
+An Istio CA is deployed in each Kubernetes namespace that enables Istio Auth.
+This CA issues certificates and keys to all pods in the same namespace.
+This approach is convenient for doing experiments.
 Because each Istio CA is scoped within a namespace, Istio CAs in different namespaces will not interfere with each other
-and they are easy to clean up through a single command.
+and they are easy to [uninstall]({{home}}/docs/tasks/installing-istio.html#uninstalling).
 
-We have the YAML files *istio-auth-X.yaml* for deploying all Istio components including Istio CA into the namespace.
-Follow [the Istio installation guide](./installing-istio.html),
-and **choose "If you would like to enable Istio Auth" in step 3**.
+## Before you begin
 
-### Option 2: (recommended) using per-cluster CA
+This task assumes you have:
 
-Only a single Istio CA is deployed for the Kubernetes cluster, in a dedicated namespace.
-Doing this offers the following benefits:
+* Disabled Istio Auth (if you enabled it) for all namespaces in the Kubernetes cluster.
+Otherwise, this process will deploy a CA that conflicts with existing CAs.
 
-* In the near future, the dedicated namespace will use
-[Kubernetes RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) (beta in Kubernetes V1.6) to provide security
-boundary. This will offer strong security for Istio CA.
+* Completed steps 1 - 3 in [the Istio installation guide]({{home}}/docs/tasks/installing-istio.html#installing-on-an-existing-cluster).
```

Comment:
If they have installed istio-auth.yaml, they do not need to do anything from here.